rubicon_ml/repository/base.py

```diff
@@ -83,7 +83,7 @@ def _ls(self, path):
 
     def _mkdir(self, dirpath):
         """Creates a directory `dirpath` with parents."""
-        return self.filesystem.mkdir(dirpath, parents=True, exist_ok=True)
+        return self.filesystem.mkdirs(dirpath, exist_ok=True)
```

Comment:
can you explain why we're changing this here just in case we need to come back and check why

Reply:
Sure! So `fsspec`'s `mkdir` command takes in a `create_parents` kwarg and not a `parents` kwarg, and at the same time it does not take in an `exists_ok` kwarg which we need it too. `fsspec`'s `mkdirs` command, however, does take in an `exist_ok` kwarg and it is noted in the `mkdir` method docs that if `create_parents = True` then 'mkdir' is equivalent to 'mkdirs' which will "create parents" like we want without needing to pass in a `create_parents` kwarg. So we make the change to `mkdirs` instead of `mkdir`.